components/style/values/specified/length.rs

```diff
@@ -18,6 +18,7 @@ use style_traits::ToCss;
 use style_traits::values::specified::AllowedNumericType;
 use super::{Angle, Number, SimplifiedValueNode, SimplifiedSumNode, Time};
 use values::{Auto, CSSFloat, Either, FONT_MEDIUM_PX, HasViewportPercentage, None_, Normal};
+use values::ExtremumLength;
```

Comment:
I defined it in https://github.com/servo/servo/pull/14740/files#diff-3f9e0fea7497b1c3fd6c91a4288f32ae.